src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertConcatenationToInterpolatedStringRefactoringProvider.cs

```diff
@@ -154,6 +155,37 @@ private static Task<Document> UpdateDocumentAsync(Document document, SyntaxNode
                         content.Add(generator.InterpolatedStringText(generator.InterpolatedStringTextToken(textWithoutQuotes)));
                     }
                 }
+                else if (syntaxFacts.IsInterpolatedStringExpression(piece) &&
+                    syntaxFacts.IsVerbatimInterpolatedStringExpression(piece) == isVerbatimStringLiteral)
+                {
+                    syntaxFacts.GetPartsOfInterpolationExpression(piece, out var _, out var contentParts, out var _);
+                    foreach (var contentPart in contentParts)
+                    {
+                        if (syntaxFacts.IsInterpolation(contentPart))
+                        {
+                            content.Add(contentPart);
+                            previousContentWasStringLiteralExpression = false;
+                        }
+                        else if (syntaxFacts.IsInterpolatedStringText(contentPart))
+                        {
+                            if (previousContentWasStringLiteralExpression)
+                            {
+                                var contentTextToken = contentPart.GetFirstToken();
+                                var newText = ConcatinateTextToTextNode(generator, content.Last(), contentTextToken.Text);
+                                content[^1] = newText;
+                            }
+                            else
+                            {
+                                content.Add(contentPart);
+                            }
+                            previousContentWasStringLiteralExpression = true;
+                        }
+                        else
+                        {
+                            Contract.Fail();
```

Comment:
either `throw ExceptionUtilities.UnexpectedValue()`; or `Debug.Fail`.

Reply:
Done in 95928ba